shopelectro/management/commands/price.py

```diff
@@ -91,6 +92,10 @@ def put_crumbs(product):  # Ignore PyDocStyleBear
             )
             return product
 
+        def put_brands(product, brands):
```

Comment:
we put brand here, not brands. So, let's name this method as `put_brand(product, brand)`